### PR TITLE
N30: Graphite

### PR DESCRIPTION
**Pull request by Keavon**
_Part of #912_

### CODE REVIEW

File: content/news/030/index.md [hunk 1]
```diff
@@ -631,6 +631,39 @@ development but, most of the basic features are implemented. That includes:
 [nodus-github]: https://github.com/r4gus/nodus
 [@r4gus]: https://github.com/r4gus
 
+### [Graphite][graphite-website]
+
+![Graphite](graphite_alpha.png)
+
+[Graphite](graphite-website) ([GitHub][graphite-repo],
```
Comment (17cupsofcoffee) [inline content/news/030/index.md:638]:
```suggestion
[Graphite][graphite-website] ([GitHub][graphite-repo],
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good! Just a couple of links that are broken, but I can fix those.

**Comment by 17cupsofcoffee** [comment]
@Keavon: I just noticed that the image in this PR is broken (I thought it was just GitHub's preview being rubbish like usual, but seeing the same thing locally), could you send me a fixed version?

**Comment by Keavon** [comment]
@17cupsofcoffee Oh no, I wonder why that is! Perhaps the `github.dev` web editor mangled the image file somehow. Here's the image: https://files.keavon.com/-/FastFineGoldenmantledgroundsquirrel/graphite_alpha.png

**Comment by Keavon** [comment]
The image showed up fine while I was editing it. I used the split-pane markdown preview and it looked great. So somehow the `github.dev` editor broke it on commit. Quite strange! I'll look out for that in the future. Thanks for your patience on this!

**Pull request merged**